### ISSUE TITLE
OrderByDoesntMatchOriginalOrderTimings is failing

### ISSUE THREAD

**Issue by grokys** [github/VisualStudio#608]
```
Test Name:  OrderByDoesntMatchOriginalOrderTimings
Test FullName:  TrackingTests.OrderByDoesntMatchOriginalOrderTimings
Test Source:    D:\projects\VisualStudio\src\TrackingCollectionTests\TrackingCollectionTests.cs : line 376
Test Outcome:   Failed
Test Duration:  0:00:00.23

Result StackTrace:  at TrackingTests.OrderByDoesntMatchOriginalOrderTimings() in D:\projects\VisualStudio\src\TrackingCollectionTests\TrackingCollectionTests.cs:line 420
Result Message: 
Expected: 1000
  But was:  0
```

I assume because it wasn't updated after #594 and we don't run that test on CI.

**Comment by StanleyGoldman** [comment]
I'll take a look at that.

**Comment by StanleyGoldman** [comment]
Wait, you mean #595

**Comment by grokys** [comment]
@StanleyGoldman yeah the timing thing fails on my machine sometimes too ;) I don't think we should be testing that in a unit test.

**Comment by paladique** [comment]
Fixed in #613, thanks @StanleyGoldman!
